pages/infrastructure_providers-validators/how_to_set_up_full_node.md

```diff
@@ -1,65 +1,125 @@
-# How to set up a full node
+# Set Up a Full Node
+Installing and running a full node allows your system to participate in a dYdX chain network.
 
-## Pre-requisite
-1. Linux (Ubuntu Server 22.04.3 recommended)
-2. 8-cpu (ARM or x86_64), 64 GB RAM, 500 GB SSD NVME Storage
+## Prerequisites
+To run a full node, the system that hosts the node must meet the following minimum requirements:
+- Linux (Ubuntu Server 22.04.3 or later recommended)
+- 8-core CPU (ARM or x86_64 architecture)
+- 64 GB RAM
+- 500 GB SSD NVMe Storage
 
-> **Note:** the example values below align with the **deployment by DYDX token holders**. For alternatives, please visit the [Network Constants page](../infrastructure_providers-network/network_constants.mdx).
+## Choose a Method
+To set up a full node, you can either:
 
-## Install Full Node
-Install a full node to participate in the network.  The following steps will guide you through the process of setting up a full node. Refer or use the script at https://github.com/dydxprotocol/v4-chain/blob/main/protocol/scripts/create_full_node.sh for a more automated process.
+1. Use [this script](https://github.com/dydxprotocol/v4-chain/blob/main/protocol/scripts/create_full_node.sh), provided by dYdX, to automate setup. 
 
-### Step 1: Update System and Install Dependencies
+Save the script with an `.sh` extension in your `$HOME` directory. You may need to replace default values, such as `chain-id`, with your own. Run it with the following commands, replacing example value `your-script-name` with your own:
```

Comment:
nit: 
1. it would be good to call out the user would need to replace the `$VERSION` ([link](https://github.com/dydxprotocol/v4-chain/blob/2eb7048e47661b7592a7ccdef436f5ed22743bf7/protocol/scripts/create_full_node.sh#L30)).

2. and we should link where users can get this info. for example, i think users can get this info from `Recommended Version` value on https://www.mintscan.io/dydx/parameters

Reply:
adding some more detail in the description

I think that too much more procedural information here will clutter a section that best serves to help readers understand their options

I do think a new section for the script would be good -- maybe its own page and table of variables /where to find yours to plug in. this top section could turn into a landing page directing users to either subpage